docs/source/install.rst

```diff
@@ -123,3 +123,35 @@ channel:
 .. _`Arch Linux` : https://www.archlinux.org/
 .. _`Hitchhiker's Guide to Python` : http://docs.python-guide.org/en/latest/
 .. _`Homebrew` : http://brew.sh
+
+Installing in ``conda`` environments
```

Comment:
Since the previous block gives the command required to install in a conda environment (possibly the root environment), what do you think about titling this section "Managing conda environments"?